lib/executor/engine/engine.cpp

```diff
@@ -109,6 +109,10 @@ Expect<void> Executor::execute(Runtime::StackManager &StackMgr,
     // Control instructions.
     case OpCode::Unreachable:
       spdlog::error(ErrCode::Value::Unreachable);
+      if (Conf.getRuntimeConfigure().isEnableCoredump()) {
+        spdlog::info("Coredump Generation taking place");
```

Comment:
Please remember to remove this line once this PR is ready.

Reply:
noted